test/helper.rb

```diff
@@ -10,18 +11,19 @@
   end
 end
 
-begin
-  require "bundler/setup"
-  # bundler/setup may not load bundler
-  require "bundler" unless Bundler.const_defined?(:ORIGINAL_ENV)
-rescue LoadError
-  warn "Failed to load bundler ... this should only happen during package building"
-end
+# begin
+#   require "bundler/setup"
+#   # bundler/setup may not load bundler
+#   require "bundler" unless Bundler.const_defined?(:ORIGINAL_ENV)
+# rescue LoadError
+#   warn "Failed to load bundler ... this should only happen during package building"
+# end
```

Comment:
Now that CI has passed, also without this section, can it be omitted altogether?

Reply:
This was the only thing blocking my merge :D